scripts/_git-all-archive.sh

```diff
@@ -116,6 +118,8 @@ rmdir_list=(
     'concurrency/docs'
     'concurrency/.ci'
     'concurrency/.circleci'
+    'networking/ThirdParty/concurrency'
+    'networking/ThirdParty/asio'
```

Comment:
did we check if any of the other libraries have submodules?

Reply:
also could probably remove the concurrency and networking tests directories as well right?

Reply:
I don't think other libraries had submodules -- if they do then they are probably required, since I think in HELICS the only special case was telling gmlc::networking to use the immediate concurrency and asio submodules from HELICS instead of the ones that are submodules of networking.

Reply:
`concurrency/tests` is already removed, line 116.